package.json

```diff
@@ -50,6 +50,7 @@
     "webpack-node-externals": "^1.5.4"
   },
   "dependencies": {
-    "hpq": "^1.1.1"
+    "hpq": "^1.1.1",
+    "lodash": "^4.17.4"
```

Comment:
Minor: If we bring this in, we'll want to think about (a) how/if we want to enforce Lodash vs. native usage (`forEach`, `assign`) and (b) if we think we'll ever need to reconcile with core's use of Underscore (maybe this always has a different build process?).